porcupine/settings.py

```diff
@@ -591,7 +591,7 @@ def _grid_widgets(
 ) -> None:
     label = ttk.Label(chooser.master, text=label_text)
     label.grid(column=0, sticky="w")
-    chooser.grid(row=label.grid_info()["row"], column=1, sticky="we")
+    chooser.grid(row=label.grid_info()["row"], column=1, sticky="we", pady=5)
```

Comment:
This `pady` does not affect checkboxes, because they don't have a separate label next to them. This means that checkboxes are a bit closer together than everything else. Ideally there would be a way to set a spacing between all items, but if there isn't anything like that, we can just add the same `pady` to checkboxes too.

Reply:
One option might be to loop through `winfo_children()` when the dialog is shown.